.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+Closes #
+
+## Changes proposed
+
+- <Briefly describe the changes made in th is pull request>
+
+## Checklist
+- [ ] The proposed changes have been reviewed and approved by a member of the documentation team.
+- [ ] The proposed changes have been thoroughly tested and verified for correctness and accuracy. 
+- [ ] I have added an entry to the ["What's New?"](https://docs.metamask.io/whats-new/) page to reflect the proposed changes. 
+    
+### Updating "What's New"
+Update the "What's New?" page with significant changes such as new features, improvements, bug fixes, and significant updates. Add entries in reverse chronological order, placing the latest updates at the top, and maintain a consistent format for each entry to enhance readability and navigation of the page.
+Follow these guidelines when adding an entry to the "What's New" page :
+    - [ ] **Title**: Provide a clear and concise title for the feature or change.
+    - [ ] **Description**: Offer a brief explanation of the change, focusing on the user or system impact.
+    - [ ] **Link**: Include a link to the PR and detailed documentation for further reading.
+   
```

Comment:
Should we qualify what counts as a major change? Also, Eric mentioned it might be nice to have a QE process or test check for some of our tutorial content. Do you think we should incorporate this into our PR template, or should we plan to address it at a later time?